okhttp/src/jvmTest/java/okhttp3/MediaTypeJavaTest.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import java.nio.charset.StandardCharsets
+import java.util.*
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.internal.platform.Platform.Companion.isAndroid
+import org.junit.jupiter.api.Test
+
+class MediaTypeJavaTest: MediaTypeGetTest() {
```

Comment:
Nit: Wanna use the suffix `Jvm` for JVM-specific Kotlin things, and `Java` for Java-language things?

Reply:
Renamed, although I probably would have called Charset related functionality Java.  But assume Java means language.

JVM = Kotlin JVM specific
Java = Java language